packages/components/src/image-cropper/stories/index.story.tsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * External dependencies
+ */
+import type { Meta, StoryObj, StoryFn } from '@storybook/react';
+/**
+ * WordPress dependencies
+ */
+import { useState, useContext } from '@wordpress/element';
+/**
+ * Internal dependencies
+ */
+import { ImageCropper, ImageCropperContext } from '../';
+import { Button, RangeControl, Flex, FlexItem } from '../../';
+
+const meta: Meta< typeof ImageCropper.Provider > = {
+	component: ImageCropper.Provider,
+	title: 'Components/ImageCropper',
+	argTypes: {
+		src: { control: { type: 'text' } },
+		width: { control: { type: 'number' } },
+		height: { control: { type: 'number' } },
+	},
+	parameters: {
+		controls: { expanded: true },
+	},
+};
+export default meta;
+
+function StateLogger() {
+	const { state } = useContext( ImageCropperContext );
+	return <pre>{ JSON.stringify( state, null, 2 ) }</pre>;
+}
+
+function TemplateControls() {
+	const { state, dispatch, getImageBlob } = useContext( ImageCropperContext );
+	const [ previewUrl, setPreviewUrl ] = useState< string >( '' );
+
+	return (
+		<>
+			<Flex justify="flex-start">
+				<FlexItem>
+					<ImageCropper />
+				</FlexItem>
+
+				{ previewUrl && (
+					<FlexItem>
+						<a href={ previewUrl } target="_blank" rel="noreferrer">
+							<img
+								src={ previewUrl }
+								alt="preview"
+								width={ state.cropper.width }
+								height={ state.cropper.height }
+							/>
+						</a>
+					</FlexItem>
+				) }
+			</Flex>
+			<Flex wrap justify="flex-start">
+				<FlexItem style={ { width: 200 } }>
+					<RangeControl
+						label="angle"
+						min={ -45 }
+						max={ 45 }
+						step={ 1 }
+						value={ state.transforms.angle }
+						onChange={ ( value ) => {
+							dispatch( {
+								type: 'ROTATE',
+								angle: Number( value ),
+							} );
+						} }
+					/>
+				</FlexItem>
+
+				<Button
+					variant="secondary"
+					onClick={ () => {
+						dispatch( {
+							type: 'ROTATE_CLOCKWISE',
+							isCounterClockwise: true,
+						} );
```

Comment:
Nit: this API is a little confusing as one onehand it's `ROTATE_CLOCKWISE` and then on the other it's `isCounterClockwise: true`. Maybe the action type could be `ROTATE_90` or something.

Reply:
Good point! I renamed it to `ROTATE_90_DEG` in [2fd0c69](https://github.com/WordPress/gutenberg/pull/63335/commits/2fd0c69b9827aec6926461ce5e45a150e66aaf74).